de.setsoftware.reviewtool.core/src/de/setsoftware/reviewtool/model/PositionTransformer.java

```diff
@@ -186,7 +186,8 @@ private static String implodePath(String[] segments, int suffixLength) {
         do {
             ret.add(curNode.path);
             curNode = curNode.next;
-        } while (curNode != null);
+        }
```

Comment:
Die Anpassung hier kommt mir komisch vor, der Zustand vorher passte zu meinem Verständnis der Code-Richtlinien.

Reply:
Dann müssen wir die checkstyle.xml anpassen, denn wenn ich das so formatiere, wie es vorher war, bekomme ich in Eclipse sofort eine Checkstyle-Warnung ("'}' an Position 9 sollte in einer eigenen Zeile stehen.").

Reply:
Eventuell benutzt Eclipse eine andere (falsche) Checkstyle-Konfiguration? Ich versuche das mal nachzuprüfen.

Reply:
OK, maven-checkstyle-plugin nutzt Checkstyle 6.18, unser Eclipse Checkstyle 8.0. Man kann es aber anpassen, dann fällt der Build auch durch, weil es eine Checkstyle-Warnung in TourCalculatorTest.java gibt, die Eclipse auch anzeigt. Ich kann für das Checkstyle-Update einen Pull-Request fertigstellen, die Frage ist aber, ob man das nicht nach den anderen Pull-Requests einarbeiten kann (sonst muss ich alle noch offenen Pull-Requests anpassen...).